packages/nimble-components/src/table/models/table-layout-manager.ts

```diff
@@ -222,10 +222,13 @@ export class TableLayoutManager<TData extends TableRecord> {
     }
 
     private resetGridSizedColumns(): void {
-        if (!this.gridSizedColumns) {
+        if (!this.gridSizedColumns || this.gridSizedColumns.length === 0) {
             return;
         }
 
+        const gridSizeColumnTotal = this.getTotalColumnFixedWidth();
```

Comment:
`this.getTotalColumnFixedWidth()` doesn't return the _grid size_ column total, right? I believe function will return the total of _all_ columns. Is that what we want here? I'm thinking it is not what we want because it would lead to fairly small fractional widths being set on the fractional width columns. Potentially all fractional width columns could have a `currentFractionalWidth` set on them of well less than `1` if there are a lot of fixed width columns.

Reply:
Oh that's right...will fix. I think this means I should write another test too.

Reply:
@mollykreis, I've now added a couple of tests to verify the existence of a fixed width column doesn't cause any unexpected behaviors. I'm not entirely happy with the descriptions I've used, nor am I sure we're covering things as completely as possibly, and I'm open to suggestions.